### ISSUE TITLE
Questions about overfitting & Q-value loss weight

### ISSUE THREAD

**Issue by return-sleep** [Zhendong-Wang/Diffusion-Policies-for-Offline-RL#15]
When I trained the model, in the middle and late stages of training, its performance gradually declined as the number of training steps increased. Is this due to overfitting the behavioral policy or is it due to exploration error? May I ask if you have encountered such a situation? Should I increase the Q-value weights or decrease them?

**Comment by Zhendong-Wang** [comment]
During my training, usually it is stable especially for mojoco tasks. Which environment are you testing? If it happens, maybe you can moniter the q-value function loss. Usually decreasing the Q-value weights stabalize the training, since the algorithm will be tuned to be more similar to a behavior cloning algorithm.

**Comment by return-sleep** [comment]
> During my training, usually it is stable especially for mojoco tasks. Which environment are you testing? If it happens, maybe you can moniter the q-value function loss. Usually decreasing the Q-value weights stabalize the training, since the algorithm will be tuned to be more similar to a behavior cloning algorithm.

Thank you for your reply， I tested on vision-based mujoco tasks and perform Diffusion_QL in latent state space. I noticed your design regarding the hyperparameter q_value (from 0.01 to 3.5 across different tasks), may I ask if is it generally true that the more complex the task is, the more we should tend to lower values, which means that we focus on the behavioral cloning term?

**Comment by Zhendong-Wang** [comment]
From my experience, it depends. If the offline dataset is already very good, then a small $eta$ could work well. If the offline dataset is just medium well, then you need more policy optimization to reach good performance.
